hooks/persistence-elastic/hook.test.js

```diff
@@ -116,3 +100,75 @@ test("should send findings to elasticsearch", async () => {
     ],
   });
 });
+
+test("should not append namespace if 'appendNamespace' is null", async () => {
+  const findings = [];
+
+  const getFindings = async () => findings;
+
+  await handle({ getFindings, scan, now, tenant: "default" });
+
+  expect(elasticClient.index).toBeCalledTimes(1);
+  expect(elasticClient.index).toBeCalledWith({
+    body: {
+      "@timestamp": now,
+      id: "09988cdf-1fc7-4f85-95ee-1b1d65dbc7cc",
+      labels: {
+        company: "iteratec",
+      },
+      name: "demo-scan",
+      parameters: ["-Pn", "localhost"],
+      scan_type: "Nmap",
+      type: "scan",
+    },
+    index: `scbv2_${now.toISOString().substr(0, 10)}`,
```

Comment:
Now that you have pinned the "now" date, we could just put the plain date string here, this would make this a lot easier to read.
I at least don't know by heart what `now.toISOString().substr(0, 10)` outputs.